metaseq/distributed/utils.py

```diff
@@ -124,10 +123,13 @@ def distributed_init(cfg: MetaseqConfig):
 
         cfg = convert_namespace_to_omegaconf(cfg)
 
+    # silence torch's distributed initialization info
+    logging.getLogger("torch.distributed.distributed_c10d").setLevel(logging.WARNING)
```

Comment:
just nit: if there was any easy way to separate it out to another file, if not, then shipit.

the distributed logs have been useful once or twice in past like for:
1) sometime debugging large distributed hangs,
2) there was a small inefficiency in Megatron code where they were init'ing PP process groups even when we are not using it, and that was slowing initializing by a lot. And I just found out only because of torch dist logs.

Reply:
Id like us to have a verbose mode where everything is logged